src/runtime_src/core/pcie/linux/system_linux.cpp

```diff
@@ -36,22 +38,32 @@
 
 namespace {
 
+xrt_core::system_linux* singleton = nullptr;
+
 // Singleton registers with base class xrt_core::system
-// during static global initialization.  If statically
+// during static global initialization. If statically
 // linking with libxrt_core, then explicit initialiation
 // is required
-static xrt_core::system_linux*
+static inline xrt_core::system_linux*
```

Comment:
inline should be compilers decision, this is definitely not critical.

Reply:
Sure, will remove 'inline'.